### PR TITLE
Small updates

### PR DESCRIPTION
**Pull request by bdunne**
Minor changes based on `rails plugin new --mountable`

### CODE REVIEW

**Comment by bdunne** [comment]
@Fryguy Please review

File: Rakefile [hunk 1]
```diff
@@ -1,15 +1,16 @@
 require 'bundler/setup'
-require 'bundler/gem_tasks'
```
Comment (Fryguy) [inline Rakefile]: Why is this removed?

**Comment by Fryguy** [comment]
LGTM :+1:

cc @durandom ... we'll probably want to this in the generator as well as the other plugin repos for consistency

**Comment by durandom** [comment]
> cc @durandom ... we'll probably want to this in the generator as well as the other plugin repos for consistency (and for keeping it as close to a clean rails plugin as possible)

except for the statistics.rake tasks this is already in the generator - but I'm collecting this stuff in my branch https://github.com/ManageIQ/manageiq/compare/master...durandom:provider_generator
